bikeshed/spec-data/readonly/languages.json

```diff
@@ -4,5 +4,7 @@
 	"es": {"name": "Spanish", "native-name": "Español"},
 	"fr": {"name": "French", "native-name": "Français"},
 	"ja": {"name": "Japanese", "native-name": "日本語"},
-	"zh-cn": {"name": "Simplified Chinese", "native-name": "简体中文"}
+	"nl": {"name": "Dutch", "native-name": "Nederlands"},
+	"zh-cn": {"name": "Simplified Chinese", "native-name": "简体中文"},
+	"zh-hans": {"name": "Simplified Chinese", "native-name": "简体中文"}
```

Comment:
I assumed these keys have to be in lowercase. Otherwise I would have spelled it "zh-Hans" and "zh-CN".